### DISCUSSION TITLE
对于 Dropdown trigger=['contextMenu'] 时的对齐点的疑惑

### DISCUSSION THREAD

**Discussion by jrr997** [ant-design/ant-design#47716] [Q&A] [answered]
为什么 Dropdown 在 `trigger={'contextMenu'}` 时, 默认情况下 overlay 是跟随鼠标的？而 hover 和 click 时 overlay 的位置是取决于 placement 的。

想要在 `trigger={'contextMenu'}`时使 placement 也生效，还得加一个属性 `alignPoint={false}`。alignPoint 是没写进文档的，不过我看用着也没问题。

正常情况下，无论 trigger 属性设置成什么，placement 都应该生效才对。我看[源码](https://github.com/ant-design/ant-design/blob/c01c1b505b57ef01f9da29b185d4e9096fea1137/components/dropdown/dropdown.tsx#L194)，Dropdown 是故意的, 想知道为啥。

**Comment by afc163** [comment, marked as answer]
因为在所有操作系统下右键菜单都是跟随鼠标的。
